vault/activity_log_test.go

```diff
@@ -643,8 +643,8 @@ func TestActivityLog_API_ConfigCRUD(t *testing.T) {
 		if err != nil {
 			t.Fatalf("err: %v", err)
 		}
-		if resp != nil {
-			t.Fatalf("bad: %#v", resp)
+		if resp != nil && len(resp.Warnings) > 0 {
+			t.Logf("got warning(s): %#v", resp.Warnings)
 		}
```

Comment:
If the warning is expected, why log it?  If anything, since it's expected, shouldn't the test error out if it's absent?

Reply:
good point, in some cases we do want that. i'll audit

Reply:
it's only expected in some cases, and these helpers don't currently track the appropriate state. i can make them

Reply:
Actually, does it make sense to enforce API specifics when we are unit testing the things that are being used by the API? I would think a test like that makes more sense to verify that the API responds appropriately in some API tests.

@mgritter what are your thoughts? what was the original reason for failing if there were warnings?

Reply:
The warnings are part of the expected behavior now, and I think we should check their presence. I should have asked Bez to do this on his original PR.

Reply:
please unresolve if this doesn't fix it: fe38ff91e